core/consensus/aura/src/lib.rs

```diff
@@ -237,10 +271,40 @@ pub fn start_aura<B, C, E, I, P, SO, Error, OnExit>(
 		Arc::new(worker),
 		sync_oracle,
 		on_exit,
-		inherent_data_providers
+		inherent_data_providers,
+		accept_old_seals,
```

Comment:
Is there a reason `start_aura2` doesn't blanket `accept_old_seals = false`?

Reply:
There isn’t.